components/ca-vote.js

```diff
@@ -0,0 +1,156 @@
+/* global api */
+define(['./helpers/create-element.js', './helpers/clear-element.js', 'document-register-element'], (createElement, clearElement) => {
+    const componentName = 'ca-vote';
```

Comment:
no need to put the component name at the top, only to use it once at the bottom, think we're just refactoring that one out